python/pyspark/tests/test_worker.py

```diff
@@ -144,6 +144,15 @@ def test_with_different_versions_of_python(self):
         finally:
             self.sc.pythonVer = version
 
+    def test_reuse_worker_of_parallelize_xrange(self):
+        def get_worker_pid(input_rdd):
+            return input_rdd.map(lambda x: os.getpid()).collect()
+        rdd = self.sc.parallelize(xrange(20), 20)
```

Comment:
Not a big deal but doesn't need to be 20.

Reply:
Copy that, change the slices number to 8 cause it needs little larger than 4 to make sure this is not a flaky test.